src/windows/webview.c

```diff
@@ -477,21 +478,23 @@ static ICoreWebView2EnvironmentOptionsVtbl VTBL5 = {
 
 // Public
 
-static HMODULE webview_load_dll(void)
+static bool webview_dll_path(WCHAR *pathw, bool as_user)
 {
+	bool ok = false;
+
 	HKEY key = NULL;
-	WCHAR *dll = NULL;
-	HMODULE lib = NULL;
+	WCHAR dll[MTY_PATH_MAX] = {0};
+
+	// https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution#detect-if-a-webview2-runtime-is-already-installed
+	const wchar_t *machine_path = L"Software\\Microsoft\\EdgeUpdate\\ClientState\\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}";
```

Comment:
Nitpick: I personally dislike mixing `WCHAR` and `wchar_t` declarations and prefer to choose one or the other for the sake of better readability, though under the hood they are exactly the same size (at least on windows).

Reply:
A fair point. I'll probably `wchar_t` everything